src/implementations/twig/components/site-footer/eu-site-footer.story.js

```diff
@@ -0,0 +1,33 @@
+import { withNotes } from '@ecl/storybook-addon-notes';
+import withCode from '@ecl/storybook-addon-code';
+import { correctSvgPath } from '@ecl/story-utils';
+
+import dataCore from '@ecl/specs-component-site-footer/demo/data-core--eu';
+import logoEuMobile from '@ecl/resources-eu-logo/condensed-version/positive/logo-eu--en.svg';
+import logoEuDesktop from '@ecl/resources-eu-logo/standard-version/positive/logo-eu--en.svg';
+import footer from './site-footer.html.twig';
+import notes from './README.md';
+
+const prepareData = (data) => {
+  correctSvgPath(data);
+  data.rows[0][0][0].logo.src_mobile = logoEuMobile;
+  data.rows[0][0][0].logo.src_desktop = logoEuDesktop;
+
+  return data;
+};
+
+export default {
+  title: 'Components/Site-wide/Site footer',
+  decorators: [withCode, withNotes],
+  parameters: {
+    layout: 'fullscreen',
+    controls: {
+      disable: true,
+    },
+  },
+};
+
+export const Core = () => footer(prepareData(dataCore));
```

Comment:
for EU there are no other footer than the Core? No Harmonised?

Reply:
Yes, for me there is only Core for the EU version, see description of ECL-460

Reply:
I added the EU harmonised version